### PR TITLE
Native globbing & Improve linting

### PR DESCRIPTION
**Pull request by denysdovhan**

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,110 +1,122 @@
-var expect = require('chai').expect;
-var Plugin = require('./');
-var sysPath = require('path');
-var fs = require('fs');
+/* eslint no-undef: 0 */
```
Comment (shvaikalesh) [inline test.js]: Why?

Comment (denysdovhan) [inline test.js]: Because of `describe`, `it`, etc.

Comment (shvaikalesh) [inline test.js]: `env mocha`

File: index.js [hunk 2]
```diff
@@ -203,20 +193,36 @@ class SassCompiler {
...
+        cmd = ['cmd', '/c', `"${cmd[0]}"`].concat(cmd.slice(1));
         this.env.windowsVerbatimArguments = true;
       }
 
-      return promiseSpawnAndPipe(cmd[0], cmd.slice(1), this.env, source.data).then(d => { return {data: d}; });
+      return promiseSpawnAndPipe(cmd[0], cmd.slice(1), this.env, source.data).then(data => ({data}));
     });
   }
 
+  get getDependencies() {
```
Comment (shvaikalesh) [inline index.js]: Nice.

File: test.js (hunk 1 above)
Comment (denysdovhan) [inline test.js]: neat
didn't know about it

File: index.js [hunk 3]
```diff
@@ -253,6 +262,7 @@ SassCompiler.prototype.brunchPlugin = true;
 SassCompiler.prototype.type = 'stylesheet';
 SassCompiler.prototype.pattern = /\.s[ac]ss$/;
 SassCompiler.prototype._bin = isWindows ? 'sass.bat' : 'sass';
-SassCompiler.prototype._compass_bin = isWindows ? 'compass.bat' : 'compass'; //eslint-disable-line camelcase
+// eslint-disable-next-line camelcase
```
Comment (shvaikalesh) [inline index.js]: Maybe we should disable camelcase for the whole file? `eslint` comments in code look shitty.

Comment (denysdovhan) [inline index.js]: Totally forgot about this PR. Fixed

**Comment by denysdovhan** [comment]
Updated.

@shvaikalesh good to go?

**Comment by shvaikalesh** [comment]
LGTM

**Pull request merged**